chatterbot/adapters/io/io_json.py

```diff
@@ -3,6 +3,10 @@
 
 
 class JsonAdapter(IOAdapter):
+    """
+    The JsonAdapter class is an IO adapter that allows
+    ChatterBot to interface with Json structured databases.
```

Comment:
This is not true. The json adapter returns data in a python dictionary format which is useful for serialization and deserialization of statements. One place this could be used may be when creating a web based api. It is unlikely that this adapter would be used with a database directly.